solr/core/src/java/org/apache/solr/handler/component/TermVectorComponent.java

```diff
@@ -251,6 +252,7 @@ public void process(ResponseBuilder rb) throws IOException {
     SolrIndexSearcher searcher = rb.req.getSearcher();
 
     IndexReader reader = searcher.getIndexReader();
+    TermVectors tm = reader.termVectors();
```

Comment:
Please improve this variable name, such as use `termVectors`

Reply:
> Please improve this variable name, such as use `termVectors`

There's a `NamedList<Object> termVectors` above already. Maybe that one could be renamed to `nlTermVectors` or `termVectorsNl` or something and then the `tm` here could become `termVectors`?

Reply:
https://github.com/apache/solr/pull/1557/commits/06f9f5fcaf3d44fa165ddfbc6b7d57bb5a30289e to rename from `tm` to `termVectors`